cmd/libs/go2idl/conversion-gen/generators/tags.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators
+
+import (
+	"github.com/golang/glog"
+	"k8s.io/kubernetes/cmd/libs/go2idl/types"
+)
+
+// extractBoolTagOrDie gets the comment-tags for the key and asserts that, if
+// it exists, the value is boolean.  If the tag did not exist, it returns
+// defaultVal.
+func extractBoolTagOrDie(key string, defaultVal bool, lines []string) bool {
```

Comment:
See comment above about exposing the glog dep.  Could do.  It felt like it was a very minor detail to be exposing in go2idl.  I'd like to think more about a *real* tags API in go2idl - even this is somewhat unsatisfying, and in a bunch of places I don't use it because I need to know whether the value was specified or defaulted...